From afc9e6542aa6f2d0d7986db3d8c2cb654e107443 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@penguin.cs.ucla.edu>
Date: Fri, 1 Apr 2016 16:55:21 -0700
Subject: [PATCH] Fix resize incompatibility with GTK+ master

Problem reported by Matthias Clasen (Bug#23144).
* src/xterm.c (handle_one_xevent) [USE_GTK]:
Do not call xg_frame_resized.
---
 src/xterm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/xterm.c b/src/xterm.c
index ba9bf50..7108207 100644
--- a/src/xterm.c
+++ b/src/xterm.c
@@ -7895,10 +7895,6 @@ handle_one_xevent (struct x_display_info *dpyinfo,
             /* Force a redisplay sooner or later to update the
 	       frame titles in case this is the second frame.  */
             record_asynch_buffer_change ();
-
-#ifdef USE_GTK
-          xg_frame_resized (f, -1, -1);
-#endif
         }
       goto OTHER;
 
-- 
2.7.4

